src/collections/news/2023-04-15-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Baby Bliss Bot - Experiments with AI
+slug: clusive-latest-features
+date: "2023-04-15"
```

Comment:
We completed the AI experiments recently. It might be better to use a date in late May or early June.

Reply:
I used the April date since that was about the time I started exploring StyleGAN2-ADA.  But, you are correct since the blog post discusses results.  In my case, I had everything wrapped up by May 17 and talked about it at the first tech meeting after my vacation.  I'll use the May 17 date.

But, that means changing the name of the file itself.  Let's see what happens.